kythe/go/extractors/config/extractor.go

```diff
@@ -196,24 +197,22 @@ func verifyRequiredTools() error {
 }
 
 func findConfig(configPath, repoDir string) (*ecpb.ExtractionConfiguration, error) {
+	var configReader io.Reader
 	// if a config was passed in, use the specified config, otherwise go
 	// hunt for one in the repository.
 	if configPath == "" {
 		// otherwise, use a Kythe config within the repo (if it exists)
 		configPath = filepath.Join(repoDir, kytheExtractionConfigFile)
 	}
 
-	configFile, err := os.Open(configPath)
+	configReader, err := os.Open(configPath)
 	if err != nil {
```

Comment:
Maybe only for file not found?

```
if os.IsNotExist(err) {
  …
} else if err != nil {
  return fmt.Errorf("opening config file: %v", err)
}
```

since that means the file exists but something else is bad.

Reply:
done

probably best to ax the log statement I had then